src/main/java/xyz/groundx/caver_ext_kas/kas/wallet/Wallet.java

```diff
@@ -3344,7 +3401,216 @@ public KeyStatus deleteKey(String keyId) throws ApiException {
+     * }</pre>
+     *
+     * @param krn The KAS resource name.
+     * @return KeyList
+     * @throws ApiException
+     */
+    public KeyList getKeyListByKRN(String krn) throws ApiException {
+        return getKeyListByKRN(krn, new WalletQueryOptions());
+    }
+
+    /**
+     * Retrieve a list of keys asynchronously.<br>
+     * GET /v2/key
+     *
+     * <pre>{@code
+     * ApiCallback<KeyList> callback = new ApiCallback<KeyList> callback() {
+     *   ....implement callback method.
+     * };
+     *
+     * String krn = "krn";
+     * caver.wallet.getKeyListByKRNAsync(krn, callback);
+     *
+     * }</pre>
+     *
+     * @param krn The KAS resource name.
+     * @param callback The callback to handle response.
+     * @return Call
+     * @throws ApiException
+     */
+    public Call getKeyListByKRNAsync(String krn, ApiCallback<KeyList> callback) throws ApiException {
+        return getKeyListByKRNAsync(krn, new WalletQueryOptions(), callback);
+    }
+
+    /**
+     * Retrieve a list of keys.<br>
+     * GET /v2/key
+     *
+     * <pre>{@code
+     * String krn = "krn";
+     * WalletQueryOptions options = new WalletQueryOptions();
+     * options.setSize(1);
+     *
+     * KeyList response = caver.wallet.getKeyListByKRN(krn, options);
+     * }</pre>
+     *
+     * @param krn The KAS resource name.
+     * @param options Filters required when retrieving data. `size`, `cursor`.
+     * @return KeyList
+     * @throws ApiException
+     */
+    public KeyList getKeyListByKRN(String krn, WalletQueryOptions options) throws ApiException {
+        String size = options.getSize() != null ? Long.toString(options.getSize()) : null;
+        return this.keyApi.retrieveKeys(krn, chainId, options.getCursor(), size);
+    }
+
+    /**
+     * Retrieve a list of keys asynchronously.<br>
+     * GET /v2/key
+     *
+     * <pre>{@code
+     * ApiCallback<KeyList> callback = new ApiCallback<KeyList> callback() {
+     *   ....implement callback method.
+     * };
+     *
+     * String krn = "krn";
+     * WalletQueryOptions options = new WalletQueryOptions();
+     * options.setSize(1);
+     *
+     * KeyList response = caver.wallet.getKeyListByKRN(krn, options, callback);
+     * }</pre>
+     *
+     * @param krn The KAS resource name.
+     * @param options Filters required when retrieving data. `size`, `cursor`.
+     * @param callback The callback to handle response.
+     * @return Call
+     * @throws ApiException
+     */
+    public Call getKeyListByKRNAsync(String krn, WalletQueryOptions options, ApiCallback<KeyList> callback) throws ApiException {
+        String size = options.getSize() != null ? Long.toString(options.getSize()) : null;
+        return this.keyApi.retrieveKeysAsync(krn, chainId, options.getCursor(), size, callback);
+    }
+
+    /**
+     * Retrieve the histories of fee delegation transactions.<br>
+     * You can find out the KRW and USD price of the fees at the time of sending the transaction.<br>
```

Comment:
How about adding a description that this API only works on cypress?
Otherwise, it would be good to include logic to verify chainId.
However, it seems necessary to check whether this temporarily supports only cypress.